framework_tvb/tvb/core/services/import_service.py

```diff
@@ -38,19 +39,24 @@
 import json
 import shutil
 from cgi import FieldStorage
+from collections import OrderedDict
 from datetime import datetime
 from cherrypy._cpreqbody import Part
 from sqlalchemy.orm.attributes import manager_of_class
 from sqlalchemy.exc import IntegrityError, SQLAlchemyError
 from tvb.basic.profile import TvbProfile
 from tvb.basic.logger.builder import get_logger
 from tvb.config.algorithm_categories import UploadAlgorithmCategoryConfig
+from tvb.config.init.introspector_registry import IntrospectionRegistry
```

Comment:
A circular dependency is introduced here, thus tvb_importer won't be part of introspection and the import of Default_Project at startup won't work.

Reply:
Correct. My suggestion to put the new dictionary on the `IntrospectionRegistry` does not work due to dependency chain.
But also the recent fix to put it on FilesHelper does not make much sense, from a logical perspective does not belong there, in my opinion.
How about putting it under tvb.config (maybe directly in init.py, as an empty dict first, then to be filled at introspection)?